app/views/docs/command-line.phtml

```diff
@@ -163,7 +131,161 @@ brew install --HEAD appwrite</code></pre>
+        </tr>
+        <tr>
+            <td><code>events</code></td>
+            <td>Array of strings</td>
+            <td>Events that trigger the function to execute. <a href="docs/events">Learn more about events</a>.</td>
+        </tr>
+        <tr>
+            <td><code>schedule</code></td>
+            <td>string</td>
+            <td>Execution schedule for the function as a <a href="https://en.wikipedia.org/wiki/Cron" target="_blank" rel="noopener">cron expression</a>.</td>
+        </tr>
+        <tr>
+            <td><code>timeout</code></td>
+            <td>int</td>
+            <td>Execution timeout of the function in seconds, with a maximum configurable limit of 900 seconds.</td>
+        </tr>
+        <tr>
+            <td><code>variables</code></td>
+            <td>JSON object</td>
+            <td>Variables provided to the function on execution stored as a key-value JSON object.</td>
+        </tr>
+    </tbody>
+</table>
+
+<h3><a href="/docs/command-line#collection-options" id="collection-options">Collection Options</a></h3>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 150px">Option</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>$id</code></td>
+            <td>string</td>
+            <td>Collection ID.</td>
+        </tr>
+        <tr>
+            <td><code>name</code></td>
+            <td>string</td>
+            <td>Collection Name.</td>
+        </tr>
+        <tr>
+            <td><code>databaseId</code></td>
+            <td>string</td>
+            <td>ID of the database to which this collection belongs.</td>
+        </tr>
+        <tr>
+            <td><code>enabled</code></td>
+            <td>bool</td>
+            <td>If the collection is enabled.</td>
+        </tr>
+        <tr>
+            <td><code>documentSecurity</code></td>
+            <td>bool</td>
+            <td>If access permissions can be defined at the document level.</td>
+        </tr>
+        <tr>
+            <td><code>permissions</code></td>
+            <td>Array of strings</td>
+            <td>Grants access permissions to users at the collection level. <a href="docs/rest#permissions">Learn more about permissions</a>.</td>
+        </tr>
+        <tr>
+            <td><code>attributes</code></td>
+            <td>Array of objects</td>
+            <td>Defines a list of attributes in the collection. <a href="docs/models/attributeList">Learn more about the Attributes List Object</a>.</td>
+        </tr>
+        <tr>
+            <td><code>indexes</code></td>
+            <td>Array of objects</td>
+            <td>Defines a list of indexes in the collection. <a href="docs/models/indexList">Learn more about the Indexes List Object</a>.</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
Wow, I was worried about how to organize this data best, but this looks great!

Reply:
These happened to work great :D